features/step_definitions/kata.rb

```diff
@@ -1,229 +1,111 @@
+  #@result = user.cli_exec(:apply, f: "deployment.yaml")
+  step %Q|I obtain test data file "kata/namespace.yaml"|
+  @result = user.cli_exec(:apply, f: "namespace.yaml")
+  raise "Failed to install sandboxed-operator" unless @result[:success]
+  step %Q|I obtain test data file "kata/operatorgroup.yaml"|
+  @result = user.cli_exec(:apply, f: "operatorgroup.yaml")
+  raise "Failed to install sandboxed-operator" unless @result[:success]
+  step %Q|I obtain test data file "kata/subscription.yaml"|
+  @result = user.cli_exec(:apply, f: "subscription.yaml")
+  raise "Failed to install sandboxed-operator" unless @result[:success]
 end
 
-Given /^the kata-operator is installed(?: to #{OPT_QUOTED})? using OLM(?: (CLI|GUI))?$/ do | kata_ns, install_method |
-  ensure_admin_tagged
-  kata_ns ||= "openshift-sandboxed-containers-operator"
-  kata_config_name = "example-kataconfig"
-  step %Q/I store master major version in the :master_version clipboard/
-  raise "Kata operator OLM installation only supported for OCP >= 4.8" unless cb.master_version >= "4.8"
-  install_method ||= 'CLI'
-  # first check pre-req
-  step %Q/I switch to cluster admin pseudo user/
-  project('openshift-marketplace')
-  unless catalog_source('qe-app-registry').exists?
-    logger.info("Kata installation depends on `qe-app-registry`, which is missing in this cluster, calling step to create it...")
-    step %Q/I create "qe-app-registry" catalogsource for testing/
+Then /^operator should be up and running$/ do
+  step %Q/I wait until sandboxed operator is ready/
+end
+
+Given /^I wait until sandboxed operator is ready$/ do
+  timeout = 120
+  kata_ns = "openshift-sandboxed-containers-operator"
+  expected_status = "Installed"
+  expected_state = "Succeeded"
+  operator_status = {
+    instruction: "Wait till operator is installed to all worker nodes",
+    success: false,
+  }
+  operator_status[:success] = wait_for(timeout, stats: {}) do
+    @result = admin.cli_exec(:get, resource: "operators", o: "jsonpath=''{..status.components.refs[4].conditions[0].type}'", n:kata_ns)[:stdout].to_s.include? expected_status
+    @result = admin.cli_exec(:get, resource: "operators", o: "jsonpath=''{..status.components.refs[5].conditions[0].type}'", n:kata_ns)[:stdout].to_s.include? expected_state
   end
+  raise "Failed to install sandboxed operator" unless @result
+  logger.info("Sandboxed operator installation status is #{expected_status}")
+end
+
+When /^I apply #{QUOTED} in #{QUOTED} namespace$/ do |kata_config_name, kata_ns|
+  step %Q/I store master major version in the :master_version clipboard/
+  step %Q|I obtain test data file "kata/release-#{cb.master_version}/kataconfiguration_v1_kataconfig.yaml"|
+  @result = user.cli_exec(:apply, f: 'kataconfiguration_v1_kataconfig.yaml', n:kata_ns)
+  raise "Failed to apply #{kata_config_name} kataconfig" unless @result[:success]
+end
 
-  unless kata_config(kata_config_name).exists?
-    if install_method == 'GUI'
```

Comment:
@valtair1891 you have taken out the installation via GUI approach in your refactor w/o a replacement.  Is that what you want?  The GUI installation testcase will need to be updated.

Reply:
For now, i'm trying to simplify everything, once 1 method (CLI) will work, i'll add another one